gateway/modules/crud/dataloader.go

```diff
@@ -145,7 +145,8 @@ func (graph *Module) dataLoaderBatchFn(c context.Context, keys dataloader.Keys)
 				defer wg.Done()
 
 				// Execute the query
-				res, err := graph.crud.Read(ctx, req.DBType, graph.project, req.Col, &req.Req)
+				req.Req.Options.Prefix = ""
```

Comment:
Add a comment on why this line is so important. Also put a note to not remove this